pkg/daemon/ceph/client/keyring.go

```diff
@@ -31,7 +31,7 @@ const (
 	// AdminKeyringTemplate is a string template of Ceph keyring settings which allow connection
 	// as admin. The key value must be filled in by the admin auth key for the cluster.
 	AdminKeyringTemplate = `
-[client.admin]
+[client.rookoperator]
```

Comment:
Need to format the client name
```suggestion
[%s]
```